runtime/src/main/java/org/corfudb/runtime/BootstrapUtil.java

```diff
@@ -35,12 +37,51 @@
      * @param retries      Number of retries to bootstrap each node before giving up.
      * @param retryTimeout Duration between retries.
      */
-    public static void bootstrap(@Nonnull Layout layout,
+    public static void bootstrap(@NonNull Layout layout,
```

Comment:
Do you want lombok's or javax?

Reply:
I was following https://github.com/CorfuDB/CorfuDB/pull/1368 to move to lombok.
And the lombok annotation adds a null checker the parameter.